diff --git a/node_modules/app-builder-lib/out/macPackager.js b/node_modules/app-builder-lib/out/macPackager.js
index e0b51e3..20d2d77 100644
--- a/node_modules/app-builder-lib/out/macPackager.js
+++ b/node_modules/app-builder-lib/out/macPackager.js
@@ -104,6 +104,22 @@ class MacPackager extends platformPackager_1.PlatformPackager {
                     outAppPath: path.join(appOutDir, appFile),
                     force: true,
                 });
+
+                const packContext = {
+                    appOutDir,
+                    outDir,
+                    arch,
+                    targets,
+                    packager: this,
+                    electronPlatformName: platformName,
+                };
+
+                await this.info.afterPack(packContext)
+
+                if (framework.afterPack != null) {
+                    await framework.afterPack(packContext)
+                }
+
                 await fs.rm(x64AppOutDir, { recursive: true, force: true });
                 await fs.rm(arm64AppOutPath, { recursive: true, force: true });
                 await this.doSignAfterPack(outDir, appOutDir, platformName, arch, platformSpecificBuildOptions, targets);
